src/jit/compiler.h

```diff
@@ -3024,6 +3024,8 @@ class Compiler
 #if FEATURE_HW_INTRINSICS
     InstructionSet lookupHWIntrinsicISA(const char* className);
     NamedIntrinsic lookupHWIntrinsic(const char* methodName, InstructionSet isa);
+    instruction insOfHWIntrinsic(NamedIntrinsic intrinsicID, var_types baseType);
+    ssize_t ivalOfHWIntrinsic(NamedIntrinsic intrinsicID);
```

Comment:
It looks like these 2 functions should be static. Also, it's not clear why they're members of `Compiler`. At least `insOfHWIntrinsic` should be moved to `CodeGen` probably.

Reply:
Several of these methods could probably be static and could likely be not in the compiler.

I put them here for consistency with the other existing methods that are driven off the `HWIntrinsicInfo` table I extended.

Reply:
It looks like the rest of the methods are related to intrinsic importation so they naturally belong here (for the lack of a better place). Consistency is usually a good thing, when the existing code isn't a mess. Unfortunately for us, the JIT code is a mess and attempting to be consistent with it is not always a useful goal.

Reply:
Moved them to be static functions defined in `hwintrinsiccodegenxarch`.